### PR TITLE
fix(schematics): added simpleModuleName for lib

### LINKED ISSUE #573: Module name is not customizable
`ng g lib feature-chart-viewer --directory=chart`

Creates the module name as `ChartFeatureChartViewer` (not a better name for module). Provide option to customize the module name at least provide option for workspace specific schematic.

### LINKED ISSUE #874: Correct command for creating a lib in libs/[subfolder]?
I want to create a dumb module inside libs/components

I run 

```
ng g lib dumb --directory components --unit-test-runner jest -d
```
I get:
```
CREATE libs/components/dumb/tsconfig.lib.json (754 bytes)
CREATE libs/components/dumb/tslint.json (254 bytes)
CREATE libs/components/dumb/src/index.ts (63 bytes)
CREATE libs/components/dumb/src/lib/components-dumb.module.ts (259 bytes)
CREATE libs/components/dumb/src/lib/components-dumb.module.spec.ts (467 bytes)
CREATE libs/components/dumb/jest.config.js (148 bytes)
CREATE libs/components/dumb/tsconfig.spec.json (264 bytes)
CREATE libs/components/dumb/src/test-setup.ts (30 bytes)
UPDATE angular.json (65618 bytes)
UPDATE nx.json (2054 bytes)
UPDATE tsconfig.json (3080 bytes)
```

It creates components-dumb.module.ts, but I want dumb.module.ts

What would be the correct command for this?

### PR DESCRIPTION
**Pull request by listepo-alterpost**
Fix https://github.com/nrwl/nx/issues/573
Fixes https://github.com/nrwl/nx/issues/874

### CODE REVIEW

**Review by vsavkin — CHANGES_REQUESTED** [review]
It looks good. One thing I am thinking is to change the name of the property from skipModulePrefix to simpleModuleName. The word "skip" implies an _action_ whereas "simple" implies a _choice_.

**Comment by listepo-alterpost** [comment]
@vsavkin done

File: packages/schematics/src/collection/library/schema.json [hunk 1]
```diff
@@ -34,6 +34,11 @@
       "type": "boolean",
       "default": false
     },
+    "simpleModuleName": {
+      "description": "Skip module prefix when using --directory",
```
Comment (FrozenPandaz) [inline packages/schematics/src/collection/library/schema.json]: One last remnant.
```suggestion
      "description": "Keep the module name simple (when using --directory)",
```

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]

**Comment by listepo-alterpost** [comment]
@FrozenPandaz done!

**Review by FrozenPandaz — APPROVED** [review]
LGTM! Thank you for your PR!

**Pull request merged**